release notes/v0.49.0.md

```diff
@@ -0,0 +1,175 @@
+
+The traces generation for the browser module depends on the overall `k6` traces option introduced in [v0.48.0](https://github.com/grafana/k6/releases/tag/v0.48.0). Check out the [documentation](https://grafana.com/docs/k6/latest/using-k6/k6-options/reference/#traces-output) to learn more about it.
+
+### gRPC streaming API becomes part of the k6 core [#3490](https://github.com/grafana/k6/pull/3490)
+
+With this release, gRPC's streaming API becomes part of the core's `k6/net/grpc` module. The experimental `k6/experimental/grpc` has been back-merged into the core.
+
+You could still use import `k6/experimental/grpc` for a couple of releases, but it is deprecated and will be removed in the future (planned in k6 version `v0.51.0`).
+
+The migration is straightforward. Just replace `k6/experimental/grpc` with `k6/net/grpc` in your script imports, and the code should work as before.
+
+### k6/html: Extract selection from element [#3519](https://github.com/grafana/k6/pull/3519)
+
+[`k6/html`](https://grafana.com/docs/k6/latest/javascript-api/k6-html/) has for been around for a while and allows you to search within an html document with an jquery-like API called [Selection](https://grafana.com/docs/k6/latest/javascript-api/k6-html/selection/), and also has support for the more standard [Element](https://grafana.com/docs/k6/latest/javascript-api/k6-html/element/) that represents DOM element.
+
+For a long time you get an Element from a Selection using the [`.get(index)`](https://grafana.com/docs/k6/latest/javascript-api/k6-html/selection/selection-get/), but you couldn't get back to a Selection from an Element.
+
+This happens to not be a common case, but one that in some cases requires rewrite of the code quite a lot.
+
+```javascript
+let li = http.get("https://test.k6.io").html().find("li");
+li.each(function(_, element) {
+    // here element is an element not a selection
+    // but what if for each li we want to select something more?
+    // in jquery that will be:
+   let container = $(element).closest('ul.header-icons');
+    // but what should `$` do?
+    // in a browser there is only 1 html document that you have access to
+    // in k6 though you can be working with multiple ones, so `$` can't know which one it should
+    // work against
+});
+```
+
+In order to support the above you can just use the selection, without going to element as in:
+
+```javascript
+let li = http.get("https://test.k6.io").html().find("li");
+for (; li.size() > 0; li = li.next()) {
+    let ul = li.closest('ul.header-icons'); // li here is still a selection and we iterate over it.
+}
+```
+
+This is not always possible though, and arguably isn't what most users will naturally do.
+
+Because of this we have now added a new [`.selection()`](https://grafana.com/docs/k6/latest/javascript-api/k6-html/element/element-selection/) which returns a selection for it's element.
```

Comment:
```suggestion
Because of this, we have now added a new [`.selection()`](https://grafana.com/docs/k6/latest/javascript-api/k6-html/element/element-selection/) which returns a selection for it's element.
```